### ISSUE TITLE
Accessibility: Content in BottomSheet is hidden from screen readers

### ISSUE THREAD

**Issue by fymmot** [manufont/react-swipeable-bottom-sheet#24]
There is an accessibility problem with the current implementation affecting visually impaired screen reader users.

When the bottom sheet is expanded, the attribute `aria-hidden=true` is set on the container element. This effectively hides everything from the screen reader, making it inaccessible.

### How to reproduce
- Go to the Fullscreen example (https://manufont.github.io/react-swipeable-bottom-sheet/fullscreen.html )
- Start a screen reader, e.g. VoiceOver
- Navigate to the "Open" button and activate it

### Expected behavior
- The bottom sheet is expanded and the screen reader will start reading its content.

### Actual behavior
- The bottom sheet is expanded
- The screen reader cannot find any content.

![image](https://user-images.githubusercontent.com/6492106/124740593-3ae30680-df1b-11eb-83fb-57c6d88cf7f2.png)
